feature/experimental/bgp/otg_tests/bgp_2byte_4byte_asn_policy_test/bgp_2byte_4byte_asn_policy_test.go

```diff
@@ -0,0 +1,565 @@
+		network1.Addresses().Add().SetAddress(subnetAddr1).SetPrefix(subnetLen1).SetCount(1)
+		network1.AsPath().Segments().Add().SetAsNumbers([]uint32{55000, 4400, 3300})
+
+		network2 := srcBgpPeer.V4Routes().Add().SetName("bgpNeti2")
+		network2.SetNextHopIpv4Address(ateSrc.IPv4).
+			SetNextHopAddressType(gosnappi.BgpV4RouteRangeNextHopAddressType.IPV4).
+			SetNextHopMode(gosnappi.BgpV4RouteRangeNextHopMode.MANUAL)
+		network2.Addresses().Add().SetAddress(subnetAddr2).SetPrefix(subnetLen2).SetCount(1)
+		network2.AsPath().Segments().Add().SetAsNumbers([]uint32{55000, 7700})
+		network2.Communities().Add().SetAsNumber(200).SetAsCustom(1)
+
+		network3 := srcBgpPeer.V4Routes().Add().SetName("bgpNeti3")
+		network3.SetNextHopIpv4Address(ateSrc.IPv4).
+			SetNextHopAddressType(gosnappi.BgpV4RouteRangeNextHopAddressType.IPV4).
+			SetNextHopMode(gosnappi.BgpV4RouteRangeNextHopMode.MANUAL)
+		network3.Addresses().Add().SetAddress(subnetAddr3).SetPrefix(subnetLen3).SetCount(1)
+
+	} else {
+		srcBgpPeer := srcBgp.Ipv6Interfaces().Add().SetIpv6Name(srcIpv6.Name()).Peers().Add().SetName(ateSrc.Name + ".BGP6.peer")
+		srcBgpPeer.SetPeerAddress(ateParams.peerIP).SetAsNumber(ateParams.localAS)
+		if connectionType == connInternal {
+			srcBgpPeer.SetAsType(gosnappi.BgpV6PeerAsType.IBGP)
+		} else {
+			srcBgpPeer.SetAsType(gosnappi.BgpV6PeerAsType.EBGP)
+		}
+		if asWidth == 2 {
+			srcBgpPeer.SetAsNumberWidth(gosnappi.BgpV6PeerAsNumberWidth.TWO)
+		}
+		prefixArr1 := strings.Split(prefixV6[0], "/")
+		mask1, _ := strconv.Atoi(prefixArr1[1])
+		prefixArr2 := strings.Split(prefixV6[1], "/")
+		mask2, _ := strconv.Atoi(prefixArr2[1])
+		prefixArr3 := strings.Split(prefixV6[2], "/")
+		mask3, _ := strconv.Atoi(prefixArr3[1])
+
+		network1 := srcBgpPeer.V6Routes().Add().SetName("bgpNeti1")
+		network1.SetNextHopIpv6Address(ateSrc.IPv6).
+			SetNextHopAddressType(gosnappi.BgpV6RouteRangeNextHopAddressType.IPV6).
+			SetNextHopMode(gosnappi.BgpV6RouteRangeNextHopMode.MANUAL)
+		network1.Addresses().Add().SetAddress(prefixArr1[0]).SetPrefix(uint32(mask1)).SetCount(1)
+		network1.AsPath().Segments().Add().SetAsNumbers([]uint32{55000, 4400, 3300})
+
+		network2 := srcBgpPeer.V6Routes().Add().SetName("bgpNeti2")
+		network2.SetNextHopIpv6Address(ateSrc.IPv6).
+			SetNextHopAddressType(gosnappi.BgpV6RouteRangeNextHopAddressType.IPV6).
+			SetNextHopMode(gosnappi.BgpV6RouteRangeNextHopMode.MANUAL)
+		network2.Addresses().Add().SetAddress(prefixArr2[0]).SetPrefix(uint32(mask2)).SetCount(1)
+		network2.AsPath().Segments().Add().SetAsNumbers([]uint32{55000, 7700})
+		network2.Communities().Add().SetAsNumber(200).SetAsCustom(1)
+
+		network3 := srcBgpPeer.V6Routes().Add().SetName("bgpNeti3")
+		network3.SetNextHopIpv6Address(ateSrc.IPv6).
+			SetNextHopAddressType(gosnappi.BgpV6RouteRangeNextHopAddressType.IPV6).
+			SetNextHopMode(gosnappi.BgpV6RouteRangeNextHopMode.MANUAL)
+		network3.Addresses().Add().SetAddress(prefixArr3[0]).SetPrefix(uint32(mask3)).SetCount(1)
```

Comment:
Same comment as above.

Reply:
Added SetType(gosnappi.BgpCommunityType.MANUAL_AS_NUMBER) to network2.Communities..